clients/cli/src/main/java/org/apache/gravitino/cli/outputs/LineUtil.java

```diff
@@ -100,4 +104,27 @@ public static String capitalize(String str) {
       }
     }
   }
+
+  /**
+   * Get the default value of a column.
+   *
+   * @param defaultValue the default value expression.
+   * @return the default value as a string.
+   */
+  public static String getDefaultValue(Expression defaultValue) {
+    if (defaultValue == null
+        || defaultValue == org.apache.gravitino.rel.Column.DEFAULT_VALUE_NOT_SET) {
+      return "N/A";
+    }
+
+    if (defaultValue instanceof Literal && ((Literal<?>) defaultValue).value() != null) {
+      return ((Literal<?>) defaultValue).value().toString();
+    } else if (defaultValue instanceof FunctionExpression) {
+      return defaultValue.toString();
+    } else if (defaultValue.references().length == 0) {
+      return "N/A";
+    }
```

Comment:
There is a special value for the current time as well. It would be good to include that.

Reply:
How about simply leaving the columns with no default values as empty?
Too many `N/A`s makes the table very busy.

At the same time, for a string (e.g. `varchar`) column, explicitly state that the default value is `""` if so. Empty strings are different from null ones.